storage/object/how-to/restore-an-object-from-glacier.mdx

```diff
@@ -40,6 +40,20 @@ dates:
       Your object remains available in `STANDARD` class for the duration you chose during the restore process. It will be transferred automatically back to `GLACIER` once the configured period is over. 
     </Message>
 
+## How to restore all objects in a bucket
+
+If you have numerous files in a bucket that you would like to restore, it may be more convenient to use the command line, for example with the s3cmd tool. To restore all files recursively in a bucket, you can invoke this command:
```

Comment:
```suggestion
If you have numerous files in a bucket that you would like to restore, it may be more convenient to use the command line, for example with the [s3cmd tool](/docs/tutorials/s3cmd/). To restore all files recursively in a bucket, you can invoke this command:
```